consensus/istanbul/backend/engine_test.go

```diff
@@ -238,7 +238,7 @@ func getGenesisAndKeys(n int, isFullChain bool) (*core.Genesis, []*ecdsa.Private
 	}
 
 	// generate genesis block
-	genesis := core.DefaultOttomanGenesisBlock()
+	genesis := core.DefaultGenesisBlock()
```

Comment:
There seems to be a really big different between the default ottoman genesis block's extra data and the default genesis block's.

Do you know what that difference is?

Reply:
Mostly it's the mainnet genesis validators. They are replaced below in `AppendValidatorsToGenesisBlock`